sha3-asm/src/lib.rs

```diff
@@ -20,7 +20,7 @@ extern "C" {
     /// size_t SHA3_absorb(uint64_t A[5][5], const unsigned char *inp, size_t len,
     ///                    size_t r);
     /// ```
-    pub fn SHA3_absorb(a: *mut Buffer, inp: *const u8, len: usize, r: usize) -> usize;
+    pub fn KECCAK_ASM_SHA3_absorb(a: *mut Buffer, inp: *const u8, len: usize, r: usize) -> usize;
```

Comment:
can avoid renaming by using `#[link_name = ""]`

Reply:
ah nice, didn't know this was a thing